figshare/Article.py

```diff
@@ -831,12 +839,48 @@ def pre_process_script_function(self):
     """
     Post-processing script command function.
     """
-    def post_process_script_function(self):
-        post_process_script_command = self.system_config["post_process_script_command"]
-        if (post_process_script_command != ""):
-            self.logs.write_log_in_file("info", f"Executing post-processing script: {post_process_script_command}.", True)
```

Comment:
Deleting this code results in the software no longer following the specification and the documentation. Please revert the deletion to allow the code to read the command to execute from the configuration and to output the log message. Since you've coded the script to be within this function, we need some way to tell the program we want your code to run. Therefore, I suggest the following: if the word `bagger` is read from the config file (line 835), then execute your code. If something else is read, do nothing and leave a comment/placeholder where an external script could be run in the future.